packages/indexer/src/configs/schema.graphql

```diff
@@ -212,9 +212,29 @@ type Relayer @entity {
 	perChainStats: [RelayerStatsPerChain]! @derivedFrom(field: "relayer")
 
 	"""
+	Latest activity for this relayer
+	"""
+	latestActivity: RelayerActivity @derivedFrom(field: "relayer")
```

Comment:
Won't adding this here still give us the same problem? Let's run a schema migration test